cmd2.py

```diff
@@ -1817,7 +1833,8 @@ def _current_script_dir(self):
         else:
             return None
 
-    def do__relative_load(self, file_path):
+    @with_argument_list
+    def do__relative_load(self, arglist):
```

Comment:
Random train of thought, I've been thinking about adding **do__relative_load** to the ``exclude_from_help`` list (which **do_eof** and **do_eos** are already in).  Adding a **do_*** method to this list simply excludes it from appearing in the help menu when an empty **help** command is issued.  Do you have an opinion on this?

Basically, is it useful for most end users to see this in the help menu, or would its presence be more likely to confuse them?

Reply:
I think we should exclude **do__relative_load** from the help by default. Developers can easily add it back in for their use case if they really want it.